tests/test_checkers.py

```diff
@@ -60,7 +57,7 @@ def run_a_checker(self, valids, invalids, checker, section='basic',
                     assert not valid
 
     @classmethod
-    def setUpClass(self):
+    def setup_class(self):
```

Comment:
Would these be better as pytest fixture? Based on the doc's, this is a mix of unittest and pytest, with pytest recommending to use a fixture. Then there wouldn't be this mix of unittest and pytest.

https://docs.pytest.org/en/latest/xunit_setup.html

Reply:
I have only good things to say about using fixtures :+1:

Reply:
Sounds good. I am still kind of fresh to the pytest scene which mostly means I have read pretty much none of the docs :laughing: